src/Module.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+class Module extends React.Component {
+  render() {
+    return (
+      <div key={this.props.module.title} className="semester">
+          <p id="sem01_mod01" className="button hover" >{this.props.module.title}</p>
```

Comment:
is this id accidentally there or on purpose? it looks to me like now every <p> will have the same id tag.